client/rest/src/main/java/org/opensearch/client/RestClient.java

```diff
@@ -912,7 +912,9 @@ private static Exception extractAndWrapCause(Exception exception) {
             return e;
         }
         if (exception instanceof SSLHandshakeException) {
-            SSLHandshakeException e = new SSLHandshakeException(exception.getMessage());
+            SSLHandshakeException e = new SSLHandshakeException(
```

Comment:
Minor: SSLHandshakeException is a very broad category. Can we narrow this down and use this message only when the exception cause is SunCertPathBuilderException, just to be a little more precise?